tempto-core/pom.xml

```diff
@@ -160,6 +155,19 @@
         </dependency>
 
         <!-- for testing -->
+        <dependency>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>jakarta.inject</groupId>
+            <artifactId>jakarta.inject-api</artifactId>
+            <version>2.0.1</version>
+            <scope>runtime</scope>
```

Comment:
This should be `test` scope since it's just for testing. Using `runtime` will make it transitive